### PR TITLE
Ria 3498 error submit

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3498

### Change description ###
RIA-3498_Error_Submit

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: e2e/features/pa-appeal-pay-now.feature [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    And I complete the `Legal representative details` page
+
+    Then I select `Decision without a hearing. The fee for this type of appeal is £80` for the `How do you want the appeal to be decided?` field
+    And I select `Decision with a hearing. The fee for this type of appeal is £140` for the `How do you want the appeal to be decided?` field
+    And I click the `Continue` button
+
+    Then I should see the text `Don't have Payment by Account?`
+    And I should see the `Apply to use Payment by Account` link
+
+    And I click the `Pay after submitting the appeal by card` label
```
Comment (bradsorour) [inline e2e/features/pa-appeal-pay-now.feature:71]: Any reason we are clicking this option?

File: e2e/features/pa-appeal-pay-now.feature [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    And I click the `Pay after submitting the appeal using Payment by Account` label
```
Comment (bradsorour) [inline e2e/features/pa-appeal-pay-now.feature:72]: Same as above. Any reason we are also clicking this option - seems unnecessary?

File: e2e/features/pa-appeal-pay-now.feature (hunk 1 above)
Comment (Lee365) [reply e2e/features/pa-appeal-pay-now.feature:71]: Additional click to check the selection works.

File: e2e/features/pa-appeal-pay-now.feature (hunk 2 above)
Comment (Lee365) [reply e2e/features/pa-appeal-pay-now.feature:72]: Additional click to check the selection works.

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**